Shellcheck fails on show-release-diffs.sh

```
In hack/show-release-diffs.sh line 119:
git show-branch --current ${MAIN_BRANCH} > "$WORKDIR/show-branch.txt"
                          ^------------^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean:
git show-branch --current "${MAIN_BRANCH}" > "$WORKDIR/show-branch.txt"


In hack/show-release-diffs.sh line 120:
_output_intermediate_csv ${MAIN_BRANCH} | sort -nu > "$WORKDIR/${MAIN_BRANCH}.csv"
                         ^------------^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean:
_output_intermediate_csv "${MAIN_BRANCH}" | sort -nu > "$WORKDIR/${MAIN_BRANCH}.csv"

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```

--- a/hack/show-release-diffs.sh
+++ b/hack/show-release-diffs.sh
@@ -116,8 +116,8 @@ fi
 WORKDIR=$(mktemp -d -t cnpg-XXXX)
 trap 'rm -rf "$WORKDIR"' EXIT
 
-git show-branch --current ${MAIN_BRANCH} > "$WORKDIR/show-branch.txt"
-_output_intermediate_csv ${MAIN_BRANCH} | sort -nu > "$WORKDIR/${MAIN_BRANCH}.csv"
+git show-branch --current "${MAIN_BRANCH}" > "$WORKDIR/show-branch.txt"
+_output_intermediate_csv "${MAIN_BRANCH}" | sort -nu > "$WORKDIR/${MAIN_BRANCH}.csv"
 _output_intermediate_csv "$branch" | sort -nu > "$WORKDIR/$branch.csv"
 
 # Prepare intermediate files

